types/rewards.yaml

```diff
@@ -59,27 +59,21 @@ IdealAttestationRewards:
     effective_balance:
       $ref: "./primitive.yaml#/Uint64"
       description: "validator's effective balance in gwei"
-      example: 1000000000
     head:
       $ref: "./primitive.yaml#/Int64"
       description: "Ideal attester's reward for head vote in gwei"
-      example: 2500
     target:
       $ref: "./primitive.yaml#/Int64"
       description: "Ideal attester's reward for target vote in gwei"
-      example: 5000
     source:
       $ref: "./primitive.yaml#/Int64"
       description: "Ideal attester's reward for source vote in gwei"
-      example: 5000
     inclusion_delay:
       $ref: "./primitive.yaml#/Uint64"
       description: "Ideal attester's inclusion_delay reward in gwei (phase0 only)"
-      example: 5000
     inactivity:
       $ref: "./primitive.yaml#/Int64"
       description: "Ideal attester's inactivity penalty in gwei"
-      example: 5000
```

Comment:
Examples are important for the swagger UI render. Please rebase on top of merged https://github.com/ethereum/beacon-APIs/pull/401

Reply:
Done